models/models.js

```diff
@@ -42,4 +42,20 @@ exports.fetchComments = (article_id) => {
       }
   return rows
    })
+}
+
+exports.addComment = (comment, article_id) => {
+   return db.query(`SELECT * FROM users WHERE username = $1`, [comment.author])
+   .then(({rows}) => {
+      if (rows.length){
```

Comment:
Is this logic needed?
If the username doesn't exist PSQL should throw the same error you are currently getting but from the SQL query below.